docs/ThreeLiner.md

```diff
@@ -1,6 +1,6 @@
 # Three Liner
 
-The `<pxb-three-liner>` can display up to three lines of stylized text or other ng-content. It is most commonly used within the context of a `<pxb-app-bar>` where the text can grow / shrink as the App Bar is expanded and collapsed.
+The `<pxb-three-liner>` can display up to three lines of stylized text or other ng-content. It is most commonly used within the context of a [`<pxb-app-bar>`](./AppBar.md) component where the text can grow / shrink as the App Bar is expanded and collapsed.
```

Comment:
might break if you are viewing this on storybook

Reply:
This is working for me, if you click the link here it should redirect to the correct doc: https://pxb-storybook-ng-feature.web.app/?path=/info/components-three-liner--readme

Reply:
That's good, because I think that's how we've been doing cross-linking for a while now :)